src/components/SubPage/Subpage.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import { Card, CardProps } from "antd";
+import { GatsbyImage, getImage } from "gatsby-plugin-image";
+import { Diagram } from "../../types";
+
+const {
+    container,
+    card,
+    dataTable,
+    dataRow,
+    dataLabel,
+    diagramCard,
+    diagramCardHeader,
+    diagramCardFooter,
+    diagramImage,
+    legendText,
+} = require("../../style/disease-subpage.module.css");
+
+export interface DataRow {
+    label: string;
+    value: string | React.ReactNode;
+}
+
+export interface SubpageProps {
+    data: {
+        rows: DataRow[];
+        diagrams?: Diagram[];
+        legendContent?: React.ReactNode;
```

Comment:
for clarity, I would make these all top level props, instead of grouping them in data. that way it's clear what is being sent into the component